src/app/dataset-view/additional-components/metadata-component/components/set-transform/components/queries-section/queries-section.component.scss

```diff
@@ -1,3 +1,11 @@
+:host {
+    ::ng-deep {
+        ngx-monaco-editor {
+            height: 300px;
+        }
+    }
+}
```

Comment:
This style definition repeats in several components. Can we do something to have it defined just once ?

Reply:
This merely a coincidence, I mean the same value.
This height is specific for each usage.
But I don't like how it looks either, I think it can be a parameter for a component

Reply:
Changed